### ISSUE TITLE
no-internal: Ignore internals defined in the same file

### ISSUE THREAD

**Issue by ilyub** [cartant/eslint-plugin-etc#52]
```ts
export function useSwipeable(): _SwipeableHandlers {
...
}

/**
 * @internal
 */
export interface _SwipeableHandlers {}
```

Currently no-internal rule reports _SwipeableHandlers in useSwipeable function.
I think it should not, because It is used in the same file = used internally.
This is legal usage.

I think it's enough to check that internal items are not used inside "import" statement.
